specification/wse/src/main/java/org/kaazing/specification/wse/internal/Functions.java

```diff
@@ -66,6 +66,11 @@ public static String asString(int value) {
         return Integer.toString(value);
     }
 
+    @Function
+    public static String uniqueUri(String origin) {
+        return origin + new String(uniqueId());
```

Comment:
Include charset in String constructor (ASCII).
Consider renaming to `withUniqueId(...)` or similar since it's not constrained to URI use case, even though that's what we'll use it for initially.